### DISCUSSION TITLE
What is the recommended version of gauge and dotnet plugin with .NET Core SDK 3.1.410 ?

### DISCUSSION THREAD

**Discussion by ayankumaar** [getgauge/gauge#2020] [General]
Hi I am running **.NET Core SDK 3.1.410 and VS Code1.57.0**. Can you please tell me what is the latest version of gauge and dotnet plugin recommended for this ?

I can see the latest gauge version is 1.2.1 and the latest dotnet plugin version is 0.4.1 (This supports .NET 5.0).

Also, please tell me what is the recommended gauge VS Code extension version ?

**Comment by sriv** [comment]
The latest gauge+gauge-dotnet should work - do you face any issue?
